distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -48,3 +50,28 @@ type Project_Description
              enso_project.namespace
     namespace : Text
     namespace self = self.prim_config.namespace
+
+## Returns the Enso project description for the given module.
+
+   Arguments:
+   - module: An optional module for which the Enso project description
+     will be fetched.
```

Comment:
What is the behaviour if the module is not provided?

Ah, I see in the example below - it's the 'current project'. Maybe let's note this in the `Arguments` too.

Reply:
Fixed in b9c3e744de3add8c0cd982e22266ecd6233b4538.